mpop_dataviz/viewmodemanager.cpp

```diff
@@ -272,7 +272,7 @@ void ViewModeManager::showOneAnswer(int numRows, int myRow, int myAnswer, const
 void ViewModeManager::goToScreensaver()
 {
     int totalBars = _viewBars[_viewActiveMode]->size();
-    for (int i = 0; i < totalBars; i++) {
+    for (int i = 0; i < 200; i++) {
```

Comment:
Is it possible to use a constant, instead of a "magic number"? That helps to keep the code as readable as possible.

I usually use something like a `static const int TOTAL_BARS = 200;` that's declared in the body of the function.

Reply:
Perfect I'm gonna do that